### PR TITLE
v0.13.4

### PR DESCRIPTION
**Pull request by rsumilang**
* Adding `alt_text` field to image asset forms.
* Add eme-form-block to Blocksmith.
* cms crash to white screen after paste
* Use `@triniti/schemas` v1.1.15

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,10910 +1,13 @@
 {
   "name": "@triniti/cms",
-  "version": "0.13.3",
-  "lockfileVersion": 2,
+  "version": "0.13.4",
+  "lockfileVersion": 1,
```
Comment (efjacobson) [inline package-lock.json:4]: @rsumilang did you intend to revert the lock file version?

Comment (rsumilang) [reply package-lock.json:4]: @efjacobson no, `npm up` did that automatically. if i change it back to `2` and run `npm install` again then it changes it back to `1`.

Comment (efjacobson) [reply package-lock.json:4]: @rsumilang what version of npm are you using?

Comment (rsumilang) [reply package-lock.json:4]: v10.13.0

Comment (efjacobson) [reply package-lock.json:4]: @rsumilang not node, npm. if you are using npm 7 it will use lockfileVersion 2. for ex i am using npm 7.7.0

Comment (rsumilang) [reply package-lock.json:4]: This is what I get with the `nvm use` command: `Now using node v10.13.0 (npm v6.4.1)`

Comment (efjacobson) [reply package-lock.json:4]: @rsumilang yeah you are using npm 6.4.1 - if you update npm itself to 7 it will use the new lockfileVersion

**Pull request merged**